code-collab-app/src/editorpage.js

```diff
@@ -57,6 +65,29 @@ class Editorpage extends React.Component {
     // Will probably involve calling unlockEditorCell()
     // call method from peer_data to update other peers // sending updates
     this.props.peer_data.send_cell_update(key);
+
+    // Reset cell last edited state
+    this.props.peer_data.current_cell = null;
```

Comment:
Moved this reset to the current_cell var here. It always ends up being paired with the broadcast function anyways, so I figured we may as well just group it in there.